### PR TITLE
Remove `timeout` to `_fetch` as API has changed

### PR DESCRIPTION
**Pull request by leomayleomay**
This PR is to:

* fix the broken `_fetch` method as API has changed
* fix the deprecated `Github.open_rest`

### CODE REVIEW

File: custom_download_strategy.rb [hunk 1]
```diff
@@ -103,6 +103,6 @@ def resolve_asset_id
 
   def fetch_release_metadata
     release_url = "https://api.github.com/repos/#{@owner}/#{@repo}/releases/tags/#{@tag}"
-    GitHub.open_rest(release_url)
+    GitHub.open_api(release_url)
```
Comment (robjacoby) [inline custom_download_strategy.rb]: It's actually `open_api` that has been deprecated in favour of `open_rest`

https://github.com/Homebrew/brew/blob/master/Library/Homebrew/utils/github.rb#L16-L20

Comment (leomayleomay) [reply custom_download_strategy.rb]: @robjacoby the thing is there's no such a method `Github.open_rest` any more, :)

Comment (robjacoby) [reply custom_download_strategy.rb]: Ah it looks like it's now namespaced `GitHub::API.open_rest`

Comment (leomayleomay) [reply custom_download_strategy.rb]: indeed :)

Comment (robjacoby) [reply custom_download_strategy.rb]:
```suggestion
    GitHub::Api.open_rest(release_url)
```

**Review by robjacoby — APPROVED** [review]

File: custom_download_strategy.rb [hunk 2]
```diff
@@ -103,6 +103,6 @@ def resolve_asset_id
 
   def fetch_release_metadata
     release_url = "https://api.github.com/repos/#{@owner}/#{@repo}/releases/tags/#{@tag}"
-    GitHub.open_rest(release_url)
+    GitHub::Api.open_rest(release_url)
```
Comment (brentonannan) [inline custom_download_strategy.rb:106]: This should be `GitHub::API`; I got `Error: uninitialized constant GitHub::Api` running `bin/setup` in `performance-api`.

I can't push a branch for a PR though, @robjacoby do I need to be added to a group for the camp?

Comment (leomayleomay) [reply custom_download_strategy.rb:106]: @brentonannan good catch!! I've filed a PR https://github.com/cultureamp/homebrew-perform-cli/pull/3

**Pull request merged**